xmppserver/src/main/java/org/jivesoftware/openfire/server/ServerDialback.java

```diff
@@ -309,6 +315,11 @@ public boolean authenticateDomain(OutgoingServerSocketReader socketReader, Strin
 
         log.debug( "Authenticating domain ..." );
 
+        if (!ServerDialback.isEnabled() && !ServerDialback.isEnabledForSelfSigned()) {
+            log.info("Failed to authenticate domain: Dialback has been disabled by configuration.");
```

Comment:
Very nitpicky:

```suggestion
            log.info("Failed to authenticate domain via Dialback: Dialback has been disabled by configuration.");
```

Presumably this isn't the only way that domain authentication might occur. I think it might always be last, but is that an implementation detail, or a requirement of the standards?

Reply:
Ignore. Logger is already going to know the class :facepalm: